docs/documentation/form/file.html

```diff
@@ -739,6 +749,23 @@ <h2 class="subtitle">
       </p>
     </div>
 
+    {% include anchor.html name="Limitations" %}
+
+    <div class="message is-warning">
+      <div class="message-body">
+        <p>
+          Please be aware that browser's validity message, for example when value of input is <code>required</code>, will not be visible when using <code>.file</code>
+          field.
+        </p>
+        <p>
```

Comment:
Remove this second paragraph. The implementation details are not relevant to the user.